blog/2022-07-29-paralus-quickstart-microk8s/index.md

```diff
@@ -249,16 +251,54 @@ data:
+Note down the `clusterID`
+
+#### Configuring HostAliases
+
+As this setup is being done on a local system, a public IP address or a domain is missing. Hence, there's a need to configure `spec.hostAliases` for the relay agent pod in the downloaded yaml file present in the cluster being imported.
+
+To make the configuration changes, you need the `CLUSTER-IP` address of Loadbalancer
+
+Use the command `kubectl get svc -A` and note down the IP address of `<releasename>-contour-envoy`
 
 ```bash
-5dceca49-c6cd-4a2b-b65a-f193c4fa001f.user.paralus.local
-5dceca49-c6cd-4a2b-b65a-f193c4fa001f.core-connector.paralus.local
+$ microk8s kubectl get svc -A
+NAMESPACE        NAME                       TYPE           CLUSTER-IP       EXTERNAL-IP      PORT(S)                         AGE
+default          kubernetes                 ClusterIP      10.152.183.1     <none>           443/TCP                         3d19h
+kube-system      kube-dns                   ClusterIP      10.152.183.10    <none>           53/UDP,53/TCP,9153/TCP          3d19h
+metallb-system   webhook-service            ClusterIP      10.152.183.206   <none>           443/TCP                         3d18h
+paralus          myrelease-kratos-courier   ClusterIP      None             <none>           80/TCP                          3d18h
+paralus          myrelease-postgresql-hl    ClusterIP      None             <none>           5432/TCP                        3d18h
+paralus          paralus                    ClusterIP      10.152.183.221   <none>           11000/TCP,10000/TCP,10001/TCP   3d18h
+paralus          myrelease-kratos-admin     ClusterIP      10.152.183.187   <none>           80/TCP                          3d18h
+paralus          dashboard                  ClusterIP      10.152.183.149   <none>           80/TCP                          3d18h
+paralus          prompt                     ClusterIP      10.152.183.65    <none>           7009/TCP                        3d18h
+paralus          myrelease-fluent-bit       ClusterIP      10.152.183.121   <none>           2020/TCP                        3d18h
+paralus          relay-server               ClusterIP      10.152.183.80    <none>           443/TCP                         3d18h
+paralus          myrelease-contour-envoy    LoadBalancer   10.152.183.106   192.168.14.160   80:30600/TCP,443:31855/TCP      3d18h
+paralus          myrelease-postgresql       ClusterIP      10.152.183.203   <none>           5432/TCP                        3d18h
+paralus          myrelease-contour          ClusterIP      10.152.183.146   <none>           8001/TCP                        3d18h
+paralus          myrelease-kratos-public    ClusterIP      10.152.183.64    <none>           80/TCP                          3d18h
+```
+
+In this case the `CLUSTER-IP` is 10.152.183.106.
+
+Open the downloaded yaml file and add the following lines under `spec.hostAliases` for `relay-agent` deployment.
```

Comment:
this is needed only if we are importing the same microk8s cluster, any other microk8s cluster ensure the load balancer ip is accessible

Reply:
Added the required statement.